tox.ini

```diff
@@ -28,20 +30,19 @@ deps =
     ; Needed for `BBHx` package to work with PyCBC
     git+https://github.com/titodalcanton/BBHx.git@py39-and-cleanup; sys_platform == 'linux'
     git+https://github.com/gwastro/BBHX-waveform-model.git; sys_platform == 'linux'
-conda_deps=
-    mysqlclient
-    lin: gcc_linux-64>=12.2.0
-    lin: gxx_linux-64>=12.2.0
-    ; mac doesn't work, need fix
-    ; mac: clang_osx-64
-    ; mac: clangxx_osx-64
+conda_deps =
+    c-compiler
+    cxx-compiler
     gsl
-    lapack==3.6.1
-conda_channels=conda-forge
-setenv =
-    ; Tell the linker to look for shared libs inside the temporary Conda env.
-    ; Needed to build BBHx's wheel, whick links to LAPACK.
-    LIBRARY_PATH={envdir}/lib:{env:LIBRARY_PATH:}
+    ligo-segments
```

Comment:
A couple of these are python packages that we should be picking up through the requirements. At least

ligo-segments and python-ligo-lw should probably be removed from conda_deps, no?